plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixClientModule.java

```diff
@@ -162,7 +167,24 @@ public static Properties getConnectionProperties(PhoenixConfig config)
         }
 
         PhoenixEmbeddedDriver.ConnectionInfo connectionInfo = PhoenixEmbeddedDriver.ConnectionInfo.create(config.getConnectionUrl());
-        connectionInfo.asProps().asMap().forEach(connectionProperties::setProperty);
+        ReadOnlyProps props = connectionInfo.asProps();
+        props.asMap().forEach(connectionProperties::setProperty);
+        if (connectionInfo.getKeytab() != null) {
+            Configuration conf = HBaseFactoryProvider.getConfigurationFactory().getConfiguration();
+            for (Map.Entry<String, String> entry : props) {
```

Comment:
Why is this meeded? We're setting the Kerberos specific properties below.
Perhaps add a comment here.